src/main/resources/swagger.api/paths/externalUser/externalUser.yaml

```diff
@@ -0,0 +1,17 @@
+get:
+  description: Get user by userId and tenantId
+  operationId: getExternalUser
+  tags:
+    - getExternalUser
+  parameters:
+    - $ref: '../../parameters/externalUserId.yaml'
+    - $ref: '../../parameters/tenantId.yaml'
+  responses:
+    '200':
+      description: 'User'
+      content:
+        application/json:
+          schema:
+            $ref: '../../schemas/response/user/user.json'
```

Comment:
Actually, this endpoint should return the same thing that it receives from `mod-users`, which is a user collection. Sorry if the description in the story was confusing. `edge-patron` shouldn't notice the difference between calling this endpoint and `mod-user`'s, in both cases it should get the same response structure.

Reply:
changed logic and implementation